### DISCUSSION TITLE
INAV Configurator 3.0-RC2

### DISCUSSION THREAD

**Discussion by DzikuVx** [iNavFlight/inav-configurator#1283] [General]
This version of INAV Configurator is intended to be used __only__ with [INAV firmware 3.0 families](https://github.com/iNavFlight/inav/releases). _INAV 2.6 and below_ are not supported at all!

# Very important!!

> When updating firmware from 2.6 to 3.0 it is advised not to restore a full diff. Defaults on multiple settings have changed! It is suggested to tune based on the new defaults!

# Known issues
* none

<table>
  <tbody>
    <tr>
      <td><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/82/Telegram_logo.svg/1024px-Telegram_logo.svg.png" width="48"></td>
      <td><a href="https://t.me/INAVFlight">INAV Official on Telegram</a></td>
    </tr>
    <tr>
      <td><img src="https://upload.wikimedia.org/wikipedia/commons/c/cd/Facebook_logo_%28square%29.png" width="48"></td>
      <td><a href="https://www.facebook.com/groups/INAVOfficial">INAV Official on Facebook</a></td>
    </tr>
  </tbody>
</table>

Support developers for free by checking out from the following stores after having clicked on the following links:

<table>
  <tbody>
    <tr>
      <td><img src="https://lh3.googleusercontent.com/TiHXyUiZ2COk7OmceBgo1qeRN2APAjWL5qUydGc-U3LqkJb3n13EhYEJ8Dpz_IACNHU" width="55"></td>
      <td><a href="https://inavflight.com/shop/u/bg">Support us on Banggood</a></td>
    </tr>
  </tbody>
</table>

## Important notes

* always read [README.md](https://github.com/iNavFlight/inav-configurator/blob/master/README.md)
* users are advised **not to use Chrome App**! Chrome Apps are deprecated and are in a process of being removed from Chrome

### Flight controllers
* [Matek F722-SE](https://inavflight.com/shop/p/MATEKF722SE)
* [Matel F722-WPS](https://inavflight.com/shop/s/bg/1778068)
* [Matek F411-WSE](https://inavflight.com/shop/s/bg/1546243)
* [Matek F765-WING](https://inavflight.com/shop/s/bg/1557661)
* [Matek F722-MiniSE](https://inavflight.com/shop/s/bg/1707404)
* [Holybro Kakute F7](https://inavflight.com/shop/s/bg/1315723)

### Airplane models
* [AR Wing Pro](https://inavflight.com/shop/s/bg/1756841)
* [AtomRC Dolphin](https://inavflight.com/shop/s/bg/1758430)
* [C1 Chaser](https://inavflight.com/shop/s/bg/1102080)
* [SonicModell Binary](https://inavflight.com/shop/s/bg/1473014)
* [Volantex Ranger 2000](https://inavflight.com/shop/s/bg/1151700)
* [Mini AR Wing](https://inavflight.com/shop/s/bg/1341528)

### Radios
* [Radiomaster TX16S](https://inavflight.com/shop/s/bg/1746075)
* [Jumper T18 Pro](https://inavflight.com/shop/s/bg/1680526)
* [FrSky Q X7](https://inavflight.com/shop/s/bg/1196246)
* [FlySky FS-i6X](https://inavflight.com/shop/s/bg/1090406) 

### Long range radio systems
* [Radiomaster TX16S + TBS Crossfire Starter Set](https://inavflight.com/shop/s/bg/1735618)
* [ImmersionsRC Ghost](https://inavflight.com/shop/s/bg/1722555)
* [SIYI FM30](https://inavflight.com/shop/s/bg/1776430)

### GPS & Sensors
* [Beitian BN180 GPS](https://inavflight.com/shop/p/BN180)
* [Beitian BN880Q GPS+COMPASS](https://inavflight.com/shop/s/bg/1450469)
* [Beitian BN880 GPS+COMPASS ](https://inavflight.com/shop/p/BN880)
* [Matek M8Q-5883](https://inavflight.com/shop/s/bg/1337288)
* [Matek Lidar+OpFlow board](https://inavflight.com/shop/s/bg/1604930)
* [Matek Digital Airspeed Sensor](https://inavflight.com/shop/s/bg/1710131)
* [Benewake TFmini Lidar](https://inavflight.com/shop/s/bg/1449740)
* [VL53L0X Lidar](https://inavflight.com/shop/s/bg/1103114)

### FPV
* [Foxeer T-Rex FPV camera](https://inavflight.com/shop/s/bg/1756536)
* [RunCam Eagle 3 FPV camera](https://inavflight.com/shop/s/bg/1723926)
* [Caddx Ratel 2 FPV camera](https://inavflight.com/shop/s/bg/1779714)
* [Caddx Vista HD](https://inavflight.com/shop/s/bg/1625165)
* [Rush Tank Ultimate](https://inavflight.com/shop/s/bg/1707214)
* [Matek 1G3 video transmitter and receiver](https://inavflight.com/shop/s/bg/1745975)
* [Eachine Cobra X FPV goggles](https://inavflight.com/shop/s/bg/1778518)
* [Skyzone SKY04X FPV goggles](https://inavflight.com/shop/s/bg/1755006)

### Other
* [ViFly ShortSaver 2](https://inavflight.com/shop/s/bg/1788341)
* [ViFly Finer buzzer](https://inavflight.com/shop/s/bg/1329066)

You can get more suggestions following this [link](https://github.com/iNavFlight/inav/wiki/Welcome-to-INAV,-useful-links-and-products) too.

<hr /><em>This discussion was created from the release <a href='https://github.com/iNavFlight/inav-configurator/releases/tag/3.0.0-RC2'>INAV Configurator 3.0-RC2</a>.</em>

**Comment by lichtl** [comment]
Hi. Is it possible to get a Mac configurator as well?  :-)

**Comment by BenWThomas** [comment]
I, too, am eager to see RC2 for Mac.
-> I've noticed that Mac releases usually lag a couple weeks behind Windows and Linux.  Always wondered why...)

iNav App lists and launches 3.0.0 RC2, but the new supported firmwares do not show up when launching Mac 3.0.0 RC2 from iNav App.

Particularly anxious for supported FlywooF745Nano; found linked firmware here:

         https://github.com/iNavFlight/inav/releases/download/3.0.0-RC2/inav_3.0.0_FLYWOOF745NANO.hex

**Reply by romko1** [reply]
Hi DzikuVx, 
Good job. Thank you.
I'm creating my custom FW for to use an external PINIO. 
So I created my new FW v.3.1.0, but I can't set it in the given configurator. 
Can you please advise me? 
Thank you

**Comment by mxiaonian** [comment]
Hi DzikuVx,
Good job. Thank you.
We need Chinese firmware. Can you set up a separate folder for the program language to facilitate the later Sinicization. For some Chinese, it's troublesome to have no Chinese
